pkg/lib/pb/model/protos/models.proto

```diff
@@ -777,6 +777,17 @@ enum RelationFormat {
 
 }
 
+enum ObjectOrigin {
+    user = 0;
+    clipboard = 1;
+    dragAndDrop = 2;
+    import = 3;
+    webClipper = 4;
+    sharingExtension = 5;
+    useCase = 6;
+    builtin = 7;
+}
```

Comment:
I would remove camelcase in webClipper and useCase words

Reply:
Done